### PR TITLE
Release notes for Edge 1.2.0

### PR DESCRIPTION
**Pull request by kevinlind**

### CODE REVIEW

**Review by emdobrin — APPROVED** [review]
Looks good to me 👍 
Will need to merge the conflicts with core once published.

**Comment by kevinlind** [comment]
> Looks good to me 👍 Will need to merge the conflicts with core once published.

Agreed. Let's wait until the AEPCore release notes (#685) are merged before merging this PR.

**Review by emdobrin — APPROVED** [review]

**Review by addb — APPROVED** [review]

**Pull request merged**
